benchmark/main.cpp

```diff
@@ -170,7 +189,7 @@ int main(int argc, char* argv[]) {
     Flags flags;
     flags.Var(colCount, 'c', "column-count", 1, "Number of columns to use");
     flags.Var(threadCount, 't', "thread-count", 1, "Number of threads");
-    flags.Var(iterations, 'i', "iterations", 6, "Number of iterations");
+    flags.Var(iterations, 'i', "iterations", -1, "Number of iterations");
```

Comment:
If you use 0, you don't need this check iterations==-1 below.

Reply:
I'm still gonna need the check, but I can change it to 0 if you want.

Reply:
You still need the check but it would look like:
int mIterations = iterations ? iterations : max(1, (int) (1 / ITERATIONS_FACTOR / size * 8 * KiB));

Reply:
Don't really like that, it should be immediate clear iterations is a number... I changed it to 0 now.